SocialFilter/sections/specific.txt

```diff
@@ -1205,7 +1205,8 @@ babyclub.de##.social-links-container
 censor.net##.footer_theme
 diy.obi.ru##.rating-section-share
 artjuku.com,cinema-sketch.com,chiepedia.club,geek.sc##.fab-btn
-artjuku.com,knowl.jp,b-gamers.net,record-now.com,pinapopo.com,daeudaeu.com,sendaiminami-tusin.com,cinema-sketch.com,mhr-movie.com,proof0309.com,flatdmkobo.com,anime-torrent.com,trovivo.com,makuring.com,appl-y.com,taisy0.com,chiepedia.club,geek.sc##.sns-btn
+sdesignlabo.com,artjuku.com,knowl.jp,b-gamers.net,record-now.com,pinapopo.com,daeudaeu.com,sendaiminami-tusin.com,cinema-sketch.com,mhr-movie.com,proof0309.com,flatdmkobo.com,anime-torrent.com,trovivo.com,makuring.com,appl-y.com,taisy0.com,chiepedia.club,geek.sc##.sns-btn
+sdesignlabo.com###twitter-widget-0
```

Comment:
@SKEIDs 
Please use 2 rules under` ! ##.twitter-timeline + widgets.js` for these kinds of twitter widgets

Reply:
ok done